.github/workflows/ci.yaml

```
@@ -12,6 +12,7 @@ jobs:
         uses: actions/checkout@v4
       - name: Run linters
         uses: golangci/golangci-lint-action@v8
+        skip-cache: true
```

actionlint (1.7.8)

unexpected key "skip-cache" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)